src/common/transformations/include/transformations/rt_info/disable_fp16_compression.hpp

```diff
@@ -25,14 +25,18 @@ TRANSFORMATIONS_API void do_not_postpone_fp16_compression(RTMap& rt_info);
 /**
  * @ingroup ie_runtime_attr_api
  * @brief DisableFP16Compression class represents runtime info attribute that marks operation
- * as prohibitted to convert to FP16 as part of Compressed Only format.
+ * as prohibited to convert to FP16 precision and they should be inferred in the precisely original precision.
```

Comment:
When you rename the class (not in this PR) please update the comment to extend it beyond FP16 mentioning FP16 as an example.

Reply:
updated in the current PR: `represents runtime info attribute that marks operation as prohibited to convert to lower precision (e.g. to FP16) and they should be inferred precisely in the original precision`.